.template-lintrc.js

```diff
@@ -1,5 +1,9 @@
 'use strict';
 
 module.exports = {
-  extends: 'octane'
+  extends: 'octane',
+
+  rules: {
+    'no-down-event-binding': 'warn',
```

Comment:
@GavinJoyce Turns out it was a template lint error and not prettier. I decided to put this in as a `warn` so it stays on our radar and we can decide what to do at a later date. The error is to help with disability so it feels like maybe it's something we want to consider. But anyway, linting doesn't fail now and your code is back to how it was originally.